src/ai/backend/agent/agent.py

```diff
@@ -1610,7 +1610,13 @@ async def _scan_images_wrapper(self, interval: float) -> None:
         self.images = await self.scan_images()
 
     @abstractmethod
-    async def push_image(self, image_ref: ImageRef, registry_conf: ImageRegistry) -> None:
+    async def push_image(
+        self,
+        image_ref: ImageRef,
+        registry_conf: ImageRegistry,
+        *,
+        timeout: float | None | Sentinel = Sentinel.TOKEN,
```

Comment:
Why is the Sentinel type necessary?

Reply:
It is to make distinguish the parameter's state between "Not defined" and "Defined as None". aiodocker, the ultimate consumer of this parameter, takes both state into account and let sets timeout as globally defined value when callee does not fill in this parameter.